cosmos/airflow/graph.py

```diff
@@ -102,6 +104,9 @@ def create_test_task_metadata(
             task_args["select"] = f"source:{node.resource_name}"
         else:  # tested with node.resource_type == DbtResourceType.SEED or DbtResourceType.SNAPSHOT
             task_args["select"] = node.resource_name
+
+        extra_context = {f"{node.resource_type.value}_config": node.config}
```

Comment:
I'm pretty curious here, why did you add the dbt node type in the key of the config, as opposed to something like `cosmos_node_config`?

Reply:
> Since this is per task/node, wouldn't it be simpler always to have the same context identifier - and if the user is interested in the node type, they can check on the node.config

@tatiana oh yeah this makes sense, there would be less context switching required for the user using the field in the callback too. I'll make the change :) 

> Also, wouldn't it be more useful if we saved other properties of the dbt node to the context, including tags and resource type..? Perhaps we could have a property in DbtNode called context_dict to expose all the information of interest.

It'll be useful to expose all the attributes of a node, should I add that too in this PR?

Reply:
I have added the entire `DbtNode` object as a dictionary into `dbt_node_config` key as `extra_context`